website/blog/testing.md

```diff
@@ -0,0 +1,512 @@
+# Lower Test Toil With Better Local Testing
+
+TODO(mwhittaker): Write an intro paragraph. Describe, briefly, the three kinds
+of testing we'll look at, and explain how they increase in complexity. Also
+maybe give a quick refresher on Service Weaver and components?
+
+## 1. Unit Testing
```

Comment:
This is nit picking, but it would be great if we can take the same example and take it through unit testing, integration testing, randomized testing, etc. It resonates better with someone who has an app and wants to do different types of testing for it.